cores/esp8266/Updater.cpp

```diff
@@ -24,8 +24,7 @@ extern "C" {
     #include "user_interface.h"
 }
 
-extern "C" uint32_t _FS_start;
-extern "C" uint32_t _FS_end;
+#include <flash_hal.h>
```

Comment:
This one needs to be reverted because
- mock version of `flash_hal.h` needs to be loaded first, to redefine `FS_start` and `FS_end`

This is only needed in sources files
- located in `cores/esp8266/`
- which make use of `FS_start` or `FS_end`